platform/security/docs/src/main/resources/IntContents.adoc

```diff
@@ -1426,3 +1426,58 @@ The `AnonymousInterceptor` is installed by default with DDF Security Application
 . Click save.
 
 Once these configurations have been added, the AnonymousInterceptor is ready for use. Both secure and non-secure requests will be accepted by all secure DDF service endpoints.
+
+== Security IdP
+
+The DDF Security IdP application provides service provider handling that satisfies the SAML 2.0 Web SSO profile in order to support external IdPs (Identity Providers).
+
+=== Components
+[cols="3", options="header"]
+|===
+
+|Bundle Name
+|Located in Feature
+|Description
+
+|security-idp-sp
+|security-idp
+|IdP Service Provider
+
+|security-idp-server
+|security-idp
+|IdP Server
+
+|===
+
+=== Installation
+
+These bundles are not installed by default, they can be stared by installing the `security-idp` feature.
+
+=== Security IdP Service Provider
+
+The IdP client that interacts with the specified Identity Provider.
+
+==== Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> IdP Client
+. Populate IdP Metadata field with an HTTPS URL (https://), file URL (file:), or XML block to refer to desired metadata (e.g., https://localhost:8993/services/idp/login/metadata)
+
+=== Security IdP Server
+
+An internal Identity Provider solution.
+
+==== Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> IdP Server
+. Click the + next to SP Metadata to add a new entry
+. Populate the new entry with an HTTPS URL (https://), file URL (file:), or XML block to refer to desired metadata (e.g., https://localhost:8993/services/saml/sso/metadata)
+
+=== Related Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> Web Context Policy Manager
+. Under Authentication Types, set:
+.. /search=SAML|IDP
```

Comment:
This should just be an example.  We should also mention that the IDP authentication type should only be used on context paths that will be accessed with browsers by users.